workflows/epigenetics/average-bigwig-between-replicates/average-bigwig-between-replicates-tests.yml

```diff
@@ -23,8 +23,8 @@
     bin_size: 50
   outputs:
     average_bigwigs:
-      class: Collection
-      collection_type: list
+      attributes:
```

Comment:
What's the model behind this ? attributes seems kind of odd, shouldn't this nest directly under the output ?

Reply:
It should and I will implement that in follow ups but that is an enhancement - this version of the models just reflects what actually works with current planemo. I don't think anyone ever thought this through and so it is a pretty low-level mapping to the XML parsing outputs.

Reply:
Don't ask my why I put this test... I cannot remember.

Reply:
(I think I am the only one to use this type of test)